packages/docs/src/routes/docs/(qwikcity)/routing/index.mdx

```diff
@@ -238,6 +239,97 @@ export default component$(() => {
 
 > The `Link` component uses the `useNavigate()` hook [internally](https://github.com/QwikDev/qwik/blob/e452582f4728cbcb7bf85d03293e757302286683/packages/qwik-city/runtime/src/link-component.tsx#L33).
 
+### Preventing navigation
```

Comment:
I think we should add this under "experimental features" section in the docs with a big disclaimer.